engine/src/main/java/org/terasology/engine/rendering/assets/mesh/resource/VertexAttribute.java

```diff
@@ -0,0 +1,58 @@
+// Copyright 2021 The Terasology Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+package org.terasology.engine.rendering.assets.mesh.resource;
+
+import org.lwjgl.opengl.GL30;
+
+/**
+ * attribute maps a target object or a primitive data to a {@link VertexResource}
+ *
+ * @param <T> the target object
+ */
+public class VertexAttribute<T, TImpl extends T> {
+
+
```

Comment:
The newlines in this file looks a bit wonky?

Reply:
yep fixed it :D